src/main.rs

```diff
@@ -75,7 +75,7 @@ struct Args {
     command: String,
 
     /// OpenID Connect issuer URL.
-    #[clap(long)]
+    #[clap(long, default_value = "https://auth.profian.com")]
```

Comment:
Extremely minor nit: the URL crate always adds a trailing slash to the domain, so a default value of `"https://auth.profian.com/"` will better match the string representation, and might prevent someone from accidentally doing `format!("{oidc_issuer}/foo")`.